src/api/worker/search/DbFacade.js

```diff
@@ -113,6 +114,8 @@ export class DbFacade {
 								db.createObjectStore(GroupDataOS)
 								db.createObjectStore(SearchTermSuggestionsOS)
 								metaOS.createIndex(SearchIndexWordsIndex, "word", {unique: true})
+								// TODO: migrate properly
```

Comment:
Yeah, about that...
If we increade DB version rn it will nuke the search index for everyone.